### ISSUE TITLE
Wrong program name in git describe output used in many places including about dialog

### ISSUE THREAD

**Issue by juozaspo** [tenacityteam/tenacity-legacy#585]
### Guidelines

- [X] I have read the guidelines.

### Version/Commit hash

b9ad2747fb9a23b91a4b6f7ff2d3b4f8a910d6c7, 0a71cad2df2d4c4c9f0e31deadb2ffa8d151c1d7 (since merging #586)

### Describe the bug.

As `git describe` command run from source directory does not return correct program name it's output stored in `GIT_DESCRIBE` variable might be used in many places such as about dialog also causing many problems in other places too.

Since forking there're no commits tagged with the updated program name and version so such information stays unchanged as shown above.

**Edit:** As #586 were merged the wrong program version is no longer an issue so this part of text was removed as it is no longer needed.

### Expected behavior

Git describe should return correct program name (e.g. `Tenacity-3.0.x` or `3.0.x` in its output), this should be done by updating it in any new tags so a correct name would be shown in the about dialog and other places.

### OS

Ubuntu Linux 20.04 LTS

#### Screenshot of about dialog (after #586)
![About Tenacity 2021-09-16 20:05:20](https://user-images.githubusercontent.com/6626242/133655277-1cb40caf-c897-4a65-84a3-e7c0cd742802.png)

### This issue is not a duplicate

- [X] I have confirmed this issue isn't a duplicate.

**Comment by nbsp** [comment]
I'd like to point out that the program name is indeed correct.
The current version of Tenacity is `Audacity-3.0.2-800-g0f131bf`. Blame Audacity for having weird versioning. The fix isn't to retroactively change the version, or to make a development release; it's to wait until the first proper release of Tenacity, whatever it may be called.

So it's not really a bug, it's just a nuisance. I don't think it's worth keeping this issue open for.

**Comment by juozaspo** [comment]
I guess you're right. Not much can be done by changing the code if it does not involve in releasing new versions.
